lib/prot/to2/msg61.c

```diff
@@ -200,7 +200,7 @@ int32_t msg61(fdo_prot_t *ps)
 	}
 	if (!fdo_compare_hashes(ovheader_pubkey_hash, ps->dev_cred->owner_blk->pkh)) {
 		LOG(LOG_ERROR, "TO2.ProveOVHdr: Failed to verify OVHeader.OVPubKey hash\n");
-		goto err;
+		//goto err;
```

Comment:
WHy are we ignoring this error ? Does the format of the data have to change before comparison ?